docker-compose.yml

```diff
@@ -48,16 +48,15 @@ services:
       - redis
     volumes:
       - ./backend/:/code/
-    
-    
+
   flower:
     env_file:
       - ./backend/.env
     build:
       context: backend
       dockerfile: Dockerfile
     container_name: flower
-    command: celery -A celery_worker flower -l info --port=5555
```

Comment:
What's the difference btween flower and beat ?

Reply:
- Flower is the web interface to monitor Celery tasks
- Beats are periodic tasks